setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import find_packages, setup
+
+import yolapy
+
+
+setup(
+    name='yolapy',
+    version=yolapy.__version__,
```

Comment:
@zoidbergwill: How about this comment instead: https://github.com/yola/yolapy/commit/b00ed8ee1b2b61c2e67b21b66741e4db49e9260f